using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -1,150 +1,271 @@
-# Adobe Campaign Standard API reference
+# Campaign API reference
 
-## Version of the Campaign Standard extension
+---
 
-The `extensionVersion()` API returns the version of the Campaign Standard extension that is registered with the Mobile Core extension.
+### extensionVersion
 
-To get the version of the Campaign Standard extension, use the following code sample:
+Returns the running version of the AEPCampaign extension.
 
 {% tabs %}
 {% tab title="Android" %}
-### Java
 
-### Syntax
+**Syntax**
 
 ```java
 public String extensionVersion()
 ```
 
-### Example
+**Example**
 
 ```java
 Campaign.extensionVersion();
 ```
+
+{% endtab %}
+{% tab title="iOS (AEP 3.x)" %}
+
+**Syntax**
+
+```swift
+static var extensionVersion: String
+```
+
+**Example**
+
+**Objective-C**
+
+```objc
+NSString *campaignVersion = [AEPMobileCampaign extensionVersion];
+```
+
+**Swift**
+
+```swift
+let campaignVersion = Campaign.extensionVersion
+```
+
 {% endtab %}
 
-{% tab title="iOS" %}
-### Syntax
+{% tab title="iOS (ACP 1.x)" %}
+
+**Syntax**
 
-```objectivec
+```objc
 + (nonnull NSString*) extensionVersion;
 ```
 
-### Examples
+**Example**
 
-### Objective C
+**Objective-C**
 
-```objectivec
-NSLog(@"ACPCampaign version: %@", [ACPCampaign extensionVersion]);
+```objc
+NSString *campaignVersion = [ACPCampaign extensionVersion];
 ```
 
-### Swift
+**Swift**
 
 ```swift
-print("ACPCampaign version: ", ACPCampaign.extensionVersion())
+let campaignVersion = ACPCampaign.extensionVersion()
 ```
+
 {% endtab %}
 
 {% tab title="React Native" %}
-### JavaScript
 
-### Syntax
+**Syntax**
 
 ```javascript
 extensionVersion(): Promise<string>
 ```
 
-### Example
+**Example**
 
 ```javascript
 ACPCampaign.extensionVersion().then(version => console.log("AdobeExperienceSDK: ACPCampaign version: " + version));
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## resetLinkageFields
+---
+
+### registerExtension
 
-This method clears the cached rules from the previous download before triggering a rule download request to the configured Campaign server. If the current SDK privacy status is not OPT\_IN, no rules download occurs.
+Registers the Campaign extension with the Mobile Core.
```

Comment:
... the Campaign extension ..

Reply:
@emdobrin, not sure what the change should be based on your comment. Can you clarify?

Reply:
Updated